### PR TITLE
Albums component

### PR DESCRIPTION
**Pull request by MauricioCoca**

### CODE REVIEW

File: src/app/main-content/album-content/album-card/album-card.component.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<div class="album-container">
+    <div class="album-photo">
+        <!-- <img src="" alt="artist photo" /> -->
```
Comment (CatherineGomez-AD) [inline src/app/main-content/album-content/album-card/album-card.component.html]: delete the innecesary line comments

File: src/app/main-content/album-content/album-card/album-card.component.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+        <img src="{{photo}}" alt="artist photo" />
+    </div>
+    <p class="album-title">{{ title }}</p>
+    <h2 class="album-release">{{releaseDate}}</h2>
+    <!-- <p>{{duration}}</p> -->
```
Comment (CatherineGomez-AD) [inline src/app/main-content/album-content/album-card/album-card.component.html]: same here

File: src/app/main-content/album-content/album-card/album-card.component.html (hunk 1 above)
Comment (MauricioCoca) [reply src/app/main-content/album-content/album-card/album-card.component.html]: done

File: src/app/models/album.model.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import { Song } from "./song.model";
+
+export interface Album {
+    id: string;
+    title: string;
+    genre: string;
+    releaseDate: string;
+    photo: string;
+    // songs: Song[];
```
Comment (AbigailGalzinB) [inline src/app/models/album.model.ts]: why are comment this?

File: tsconfig.json [hunk 4]
```diff
@@ -19,12 +18,13 @@
         "target": "ES2022",
         "module": "ES2022",
         "useDefineForClassFields": false,
-        "lib": ["ES2022", "dom"]
+        "lib": ["ES2022", "dom"],
+        "strictPropertyInitialization": false
```
Comment (CarolinaMel-AD) [inline tsconfig.json]: why did you add this ?

**Comment by MauricioCoca** [comment]
albums-content

**Review by CarolinaMel-AD — APPROVED** [review]

File: src/app/models/album.model.ts (hunk 3 above)
Comment (MauricioCoca) [reply src/app/models/album.model.ts]: removed

**Comment by AbigailGalzinB** [comment]
Please review the files that are using 2 spaces, and don't use this: "strictPropertyInitialization": false

**Comment by MauricioCoca** [comment]
albums-component

**Review by AbigailGalzinB — APPROVED** [review]
